packages/fennel/build.sh

```diff
@@ -2,9 +2,11 @@ TERMUX_PKG_HOMEPAGE="https://fennel-lang.org"
 TERMUX_PKG_DESCRIPTION="A Lisp that compiles to Lua"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.4.2
+TERMUX_PKG_VERSION=1.5.1
 TERMUX_PKG_SRCURL="https://github.com/bakpakin/Fennel/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz"
-TERMUX_PKG_SHA256=b44a205ee7ebdee22f83d2a7a87742172295b8086b5361850dfab4f49699e44f
+TERMUX_PKG_SHA256=7456737a2e0fc17717ea2d80083cfcf04524abaa69b1eb79bded86b257398cd0
 TERMUX_PKG_DEPENDS="lua53"
```

Comment:
I'm not sure if Fennel strongly depends on the version it was compiled with.
If not you could also do:
```suggestion
TERMUX_PKG_BUILD_DEPENDS="lua53"
TERMUX_PKG_DEPENDS="lua54 | lua53 | lua52 | lua51 | luajit"
```
Provided `fennel` works with all of those versions.

Reply:
Thank you! Fennel is actually written in such a way that we can support all of Lua 5.1-5.4 and luajit from the same source - the `LUA`/`LUA_VERSION` variables only impact which which lua executable is used during the build process and what is goes in the shebang for the fennel script.

As I mentioned below, I'd marked this as something to follow up on independently of bumping the version, but this saves me the trouble. Happy to incorporate this change.

Reply:
Oh yeah that can definitely be done at a later time.
The alternatives syntax could definitely use some highlighting in the next revision of the build system docs.

It's not currently mentioned in the docs.
And it isn't used *that widely*.
[As evident by the search results for it.](https://github.com/search?q=repo%3Atermux%2Ftermux-packages+%2F.*DEPENDS.*%5C%7C.*%5B%27%22%5D%2F+path%3A%2F%5E%28%7Croot-%7Cx11-%29packages%5C%2F%2F&type=code)